app/controllers/rule_countries_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RuleCountriesController < ApplicationController
+  before_action :authorize!
+
```

Comment:
this needs to the dash to hyphen's `before_action`, just verified that `negative-rule` is not being set/updated correctly

Reply:
applied